### PR TITLE
- Create drawer menu 

### PR DESCRIPTION
**Pull request by Parthshiroya06**
Create drawer menu and inside of two screen 
- Home
- Setting

### CODE REVIEW

File: app/(drawer)/home/index.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+})
```
Comment (rockingrohit9639) [inline app/(drawer)/home/index.tsx:26]: We are using `Native Base` so you can ignore creating the stylesheet and you can give props to component itself.

so import `View` from `native-base` and inject the styling that component

File: app/login.tsx [hunk 2]
```diff
@@ -80,9 +81,10 @@ export default function Login() {
 
           <FormControl.ErrorMessage>{errors.password?.message}</FormControl.ErrorMessage>
         </FormControl>
-
-        <Button onPress={handleSubmit(handleLogin)}>Submit</Button>
-
+        <Link href={'/(drawer)/home'} style={{ marginTop: 16, fontSize: 18 }}>
+          {/* <Text>Hellosd</Text> */}
```
Comment (rockingrohit9639) [inline app/login.tsx:85]: Remove this unnecessary comment. And please try avoiding them next time.

**Review by rockingrohit9639 — CHANGES_REQUESTED** [review]
Please add the `Drawer` navigation in root `_layout` and remove other unnecessary code.

File: app/login.tsx (hunk 2 above)
Comment (Parthshiroya06) [reply app/login.tsx:85]: Hello,

I have to  apply your suggestion & create PR

**Review by rockingrohit9639 — APPROVED** [review]

**Pull request merged**